cockroachcloud/troubleshooting-page.md

```diff
@@ -65,7 +59,7 @@ FATAL: CodeParamsRoutingFailed: rejected by BackendConfigFromParams: Invalid clu
 
 <h4>Solution</h4>
 
-Check that you are using the correct cluster and database names. You can find these parameters in the {{ site.data.products.db }} Console by navigating to **Cluster Overview** > **Connect** > **Step 2. Connect** > **Connection parameters**. For most tools, the full name of your database should be in the format `<routing-id>.<database>` for {{ site.data.products.serverless }} clusters.
+Check that you are using the correct cluster and database names. You can find these parameters in the {{ site.data.products.db }} Console by navigating to **Cluster Overview** > **Connect** > **Step 2. Connect** > **Connection parameters**.
```

Comment:
Is there a spot where we do want to show/explain that there are multiple options to connect? What if the SNI version doesn't work for a GUI tool or something? Will there be somewhere we point users to?

Reply:
There isn't any tool where the user can get the old-style connection string. Recreating the old-style connection string from the new parameters would be really confusing to document.

AFAIK in the bug-bash we didn't find any tools that were incompatible with the SNI version.

IMO, if this is important, then we should add an option to the Cloud Console and `ccloud` to get the old-style connection string.